index.html

```diff
@@ -4,9 +4,146 @@
+  <meta name="author" content="Miguelangel Escorche" />
+  <script
+    src="https://kit.fontawesome.com/27e53b408d.js"
+    crossorigin="anonymous">
+  </script>
+  <link rel="icon" type="image/png" sizes="32x32" href="assets/icon/valorant-logo.svg" />
+  <link rel="stylesheet" href="./css/home/home.css">
+  <title>Valorant Master</title>
 </head>
-<body>
+<body> 
+  <div class="top-bar">
+    <i class="fab fa-facebook-f"></i>
+    <i class="fab fa-twitter"></i>
+    <p>Logout</p>
+    <p>Mypage</p>
+    <p>Spanish</p>
+  </div>  
+  <header>
+    <nav>
+      <a href="index.html" class="link">
+        <img src="assets/icon/valorant-menu-icon.svg" alt="valorant logo">
+      </a>
+      <ul class="menu ">
+          <li>
+              <a href="about.html" class="link">About</a>
+          </li>
+          <li>
+              <a href="#classify" class="link">Steps to Win</a>
+          </li>
+          <li>
+              <a href="#winners" class="link">Last Winners</a>
+          </li>
+          <li class="join-container">
+              <a href="#sponsors" class="link border">Our Sponsors</a>
+          </li>
+      </ul>
+      <i class="fas fa-bars menu-icon" id="hamburguerIcon"></i>
+    </nav>
+
+    <div id="sideBar" class="sidebar">
+      <!-- Modal content -->
+        <div class="sidebar-content">
+            <span class="close-sidebar">&times;</span>
+            <ul>
+                <li><a href="about.html" class="sidebarNavigation" >About</a></li>
+                <li><a href="#classify"  class="sidebarNavigation">Steps to Win</a></li>
+                <li><a href="#winners" class="sidebarNavigation">Last Winners</a></li>
+                <li><a href="#sponsors" class="sidebarNavigation">Our Sponsors</a></li>
+            </ul>
+        </div>
+    </div>
+
+  </header>
+  <main>
+    <section id="headline" class="headline">
+      <div class="headline-title">
+        <p>"Hello! Valorant Players"</p>
+        <h1>VCT VALORANT MASTERS TOURNAMENT 2021</h1>
+      </div>
+      <div class="headline-description">
+        <p>Let's start with Challengers. Regions around the world will compete in Challengers events in three stages throughout the year. Each stage features two Challengers main events, which any team from the region can qualify for through open qualifiers. At the main events, teams compete for a place in the Challenger qualifiers. Teams that make it to and win the Challengers qualifiers will qualify for a global competition at the Masters events.</p>
+        <h2>2021.07.15(THU) ~ 16(MON)</h2>
+        <p class="address">@ Reykjavík, Iceland, Venue: Laugardalshöll</p>
+      </div>
+    </section>
+
+    <section id="classify" class="classify-steps">
+      <h1>Step to Classify to Champions</h1>
+      <div class="divider"></div>
+      <ul class="classify-steps-cards">
+        <li>
+          <img src="./assets/icon/register.svg" alt="">
```

Comment:
- [ ] **Please provide a meaningful value to `alt` attributes**

`alt` attributes play a significant role when it comes to the accessibility of the page. Good accessibility makes it easy for your website to be accessed over a range of devices. It also helps in better ranking. 

***Please cross-check all the `<img>` tags and make sure that every one of them has a meaningful alt text.**